tests/testthat/test_model_eval.R

```diff
@@ -60,7 +60,7 @@ test_that("test do_roc with 2 numeric values", {
   )
   predicted <- prediction(model_data)
 
-  predicted[["CANCELLED"]] <- c(2, 4, 4, 2, 2, 4, 4, 2, 2, 2, 2, 4, 2, NA, 2)
+  predicted[["CANCELLED"]] <- c(NA, 4, 4, 2, 2, 4, 4, 2, 2, 2, 2, 4, 2, NA, 2) # Testing the case where the data starts with NA, which used to mess up filtering NAs in do_roc.
```

Comment:
what if it is all NA?

Reply:
data.frame with one line like the following is returned.
```
⠋ |   0   1   | test case 19# A tibble: 1 x 2
  true_positive_rate false_positive_rate
               <dbl>               <dbl>
1                  1                   1
```